### PR TITLE
chore(Release): 1.0.1

### PR DESCRIPTION
**Pull request by arriolac**
Release 1.0.1.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file. Dates are d
 
 #### [Unreleased](https://github.com/googlemaps/android-maps-utils/compare/1.0.0...HEAD)
```
Comment (barbeau) [inline CHANGELOG.md]: Should be `/1.0.1...HEAD)`

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file. Dates are d
...
 
+#### [1.0.1](https://github.com/googlemaps/android-maps-utils/compare/1.0.0...1.0.1)
+
+> 18 March 2020
+
+### Features
+* Expose PolyUtil.DEFAULT_TOLERANCE ([46cbac5](https://github.com/googlemaps/android-maps-utils/commit/46cbac53883d0c5a99492b005b18a3ea558737b1))
+
+### BREAKING CHANGES
```
Comment (barbeau) [inline CHANGELOG.md]: This BREAKING CHANGES section shouldn't be here, right? That's specific to v1.0.0.

Comment (arriolac) [reply CHANGELOG.md]: Wups! Did this too quick. 😓 Will update.

Comment (arriolac) [reply CHANGELOG.md]: Done d8c7ae4

**Pull request merged**